tests/conftest.py

```diff
@@ -48,10 +48,8 @@ def runner():
 
 @pytest.fixture
 def aardvark_record_all_fields():
-    return next(
-        JsonTransformer.parse_source_file(
-            "tests/fixtures/aardvark/aardvark_record_all_fields.jsonl"
-        )
+    return JsonTransformer.parse_source_file(
```

Comment:
Hmm, how do we feel about renaming:
`JsonTransformer` -> `JSONTransformer`--
using common capitalized abbreviations when naming functions?

Reply:
This is following the capitalization pattern of classes like `Ead` where is was decided at some point in the past that all-caps class names like `EAD` weren't preferred. Not sure if that is following some linting principle but I don't have strong feelings either way about the capitalization

Reply:
Just my two cents: I prefer, and have used, all caps like `JSONTransformer` or `SQSClient` as examples.  But I noticed Transmog follows a pretty strict CamelCasing for classes.  Doesn't feel like a deal breaker either way to me, but maybe leaning towards consistency in the project.

Reply:
Hmm, I don't think it's based on a linting rule--since it seemed pass without error in `geo-harvester`, but I do think CamelCasing is generally preferred. I'm okay with it being consistent just within the repo at the least, so it's fine as is!

Reply:
Honestly, I have to stop myself typing `JSON` rather than `Json` every time and we aren't consistent about it in other repos so I'll switch it to `JSONTransformer`